### PR TITLE
IBX-395: Updated code to Bootstrap 5

### PR DESCRIPTION
**Pull request by dew326**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/IBX-395
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | yes
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

https://github.com/ezsystems/ezplatform-personalization/pull/216
https://github.com/ezsystems/ezplatform-search/pull/16
https://github.com/ezsystems/ezplatform-page-builder/pull/794
https://github.com/ezsystems/ezplatform-connector-dam/pull/35
https://github.com/ezsystems/ezplatform-form-builder/pull/293
https://github.com/ezsystems/ezplatform-segmentation/pull/31
https://github.com/ezsystems/ezplatform-site-factory/pull/130
https://github.com/ezsystems/ezplatform-workflow/pull/199
https://github.com/ezsystems/ez-support-tools/pull/100
https://github.com/ezsystems/ezcommerce-admin-ui/pull/95
https://github.com/ezsystems/date-based-publisher/pull/230
https://github.com/ezsystems/ezplatform-admin-ui-assets/pull/28
https://github.com/ibexa/recipes/pull/35

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/admin.location.adaptive.tabs.js [hunk 1]
```diff
@@ -52,7 +52,7 @@
                     const tabLinkId = itemElement.dataset.tabLinkId;
                     const tabToShow = tabsList.querySelector(`.ibexa-tabs__link#${tabLinkId}`);
 
-                    $(tabToShow).tab('show');
+                    bootstrap.Tab.getOrCreateInstance(tabToShow).show();
```
Comment (lucasOsti) [inline src/bundle/Resources/public/js/scripts/admin.location.adaptive.tabs.js:55]: is `window.bootstrap` missing here?

File: src/bundle/Resources/public/js/scripts/admin.location.tooglecontentpreview.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-(function (global, doc, localStorage, $) {
+(function(global, doc, localStorage) {
```
Comment (tischsoic) [inline src/bundle/Resources/public/js/scripts/admin.location.tooglecontentpreview.js:1]:
```suggestion
(function(global, doc, localStorage, bootstrap) {
```

File: src/bundle/Resources/public/scss/_custom.scss [hunk 3]
```diff
@@ -247,13 +247,13 @@ $theme-colors: (
...
 // Typography
 $base-font-size: 16px;
-$small-font-size: calculateRem(14px) !default;
-$h1-font-size: calculateRem(28px) !default;
-$h2-font-size: calculateRem(22px) !default;
-$h3-font-size: calculateRem(18px) !default;
-$h4-font-size: calculateRem(16px) !default;
-$h5-font-size: calculateRem(14px) !default;
-$h6-font-size: calculateRem(12px) !default;
+$small-font-size: 14px !default;
```
Comment (tischsoic) [inline src/bundle/Resources/public/scss/_custom.scss:250]: Just question: we do not want `calculateRem` here? Why?

**Review by tischsoic — CHANGES_REQUESTED** [review]

File: src/bundle/Resources/public/scss/_custom.scss (hunk 3 above)
Comment (dew326) [reply src/bundle/Resources/public/scss/_custom.scss:250]: Boostrap is multiplying those values and it triggers an error.

**Review by lucasOsti — APPROVED** [review]

File: src/bundle/Resources/public/scss/_custom.scss (hunk 3 above)
Comment (tischsoic) [reply src/bundle/Resources/public/scss/_custom.scss:250]: What about fixing out `calculareRem`? It does not convert values properly, will fixing it fix this bootstrap problem?

Fixed`calculateRem`:
```scss
@function calculateRem($size) {
    $remSize: $size / $base-font-size;

    @return $remSize + 0rem;
}
```

Comment (dew326) [reply src/bundle/Resources/public/scss/_custom.scss:250]: discussed in private

**Review by tischsoic — APPROVED** [review]

**Pull request merged**